src/pages/Homepage/Homepage.css

```diff
@@ -0,0 +1,41 @@
+.homepage-content{
+    display: flex;
+    gap: 2rem;
+    justify-content: space-around;
+    align-items: center;
+    width: 80vw;
+    height: auto;
+    position:static;
```

Comment:
Default position is `static` so no need to write it

Reply:
i removed it in later branches